client/components/editProject/editProject.html

```diff
@@ -27,12 +27,18 @@ <h3 ng-hide="editing" class="modal-title">New Project</h3>
         <p class="help-block" ng-show="editProjectForm.githubUsername.$error.required && submitted">
           The project owner's github username is required
         </p>
+        <p class="help-block" ng-show="editProjectForm.githubUsername.$error.mongoose && submitted">
+          This field cannot have a '/' character
+        </p>
       </div>
       <div class="form-group" ng-class="{ 'has-success': editProjectForm.githubProjectName.$valid && submitted,
       'has-error': editProjectForm.githubProjectName.$invalid && submitted }">
         <p class="help-block" ng-show="editProjectForm.githubProjectName.$error.required && submitted">
           The project's github name is required
         </p>
+        <p class="help-block" ng-show="editProjectForm.githubProjectName.$error.mongoose && submitted">
```

Comment:
These two `This field cannot have a '/' character` messages shouldn't stack. Lets just use 
`ng-show="(editProjectForm.githubUsername.$error.mongoose || editProjectForm.githubProjectName.$error.mongoose) && submitted"` or something of the sort